tests/layer_tests/pytorch_tests/test_rsub.py

```diff
@@ -83,17 +83,17 @@ def forward2(self, lhs, rhs:int):
         return aten_rsub(lhs_type, rhs_type), ref_net, "aten::rsub"
 
     @pytest.mark.parametrize(("lhs_type", "rhs_type"),
-                             [[torch.int32, "int"],
-                            #  [torch.int32, "float"], fp64 produce ov error of eltwise constant fold
-                              [torch.int64, "int"],
-                            #  [torch.int64, "float"], fp64 produce ov error of eltwise constant fold
-                              [torch.float32, "int"],
-                              [torch.float32, "float"],
+                             [[torch.int32, np.int32],
```

Comment:
not sure that it is correct, rsub operator injected into model only if the second operand is a native python scalar, so it is the reason why I used native python dtypes in test

Reply:
It is still python scalar in pytorch model. This type goes to np.array function because of that we need to specify 32bit type otherwise it will be 64bit.

Reply:
True, but we currently decided to convert python native types to 32bit in frontend, because this will happen any way and we want to do it earlier, cannot say it is the best approach, but it works fine for now.